playshogi-website-gwt-mvn/src/main/java/com/playshogi/website/gwt/client/widget/problems/ProblemFeedbackPanel.java

```diff
@@ -76,16 +83,19 @@ public void onUserFinishedProblemEvent(final UserFinishedProblemEvent event) {
         }
     }
 
-    @EventHandler
```

Comment:
Why are we removing this?

Reply:
Oops, this removal was from an earlier attempt which repurposed existing text fields.  I will correct this...